src/item.cpp

```diff
@@ -104,6 +106,11 @@ Item::Item(const rapidjson::Value &json) :
     if (json.HasMember("identified") && json["identified"].IsBool())
         identified_ = json["identified"].GetBool();
 
+    if (json.HasMember("craftedMods") && json["craftedMods"].IsArray())
```

Comment:
Maybe add a check that the array isn't empty, just so that we're safe against random ggg api changes.